Tests/Shopware/Connect/ProductQueryTest.php

```diff
@@ -101,9 +101,14 @@ public function getProductQuery()
                     new MarketplaceGateway(Shopware()->Models()),
                     $this->productTranslator,
                     $this->contextService,
-                    $this->localMediaService
+                    $this->localMediaService,
+                    null,
```

Comment:
Here you don't need to use null for mediaService argument if you change their places.